go2rtc_client/ws/client.py

```diff
@@ -128,16 +134,8 @@ async def _receive_messages(self) -> None:
                         self._process_text_message(msg.data)
                     case _:
                         _LOGGER.warning("Received unknown message: %s", msg)
-        except Exception:
-            _LOGGER.exception("Unexpected error while receiving message")
-            raise
-        finally:
-            _LOGGER.debug(
-                "Websocket client connection from %s closed", self._server_url
-            )
-
-            if self.connected:
-                await self.close()
+            except Exception:  # pylint: disable=broad-except
```

Comment:
This looks like it will swallow ConnectionReset exceptions which means this could be an infinite loop

Reply:
Actually we swallow everything but `asyncio.CancelledError` and `asyncio.TimeoutError` upstream and convert it to `WSMessageError`

Reply:
And `receive` has no timeout so nothing will get swallowed so this is fine.